features/android/android_callbacks.feature

```diff
@@ -38,3 +38,80 @@ Feature: Callbacks
         And the session payload field "device.osVersion" equals "Custom OsVersion"
         And the session payload field "device.runtimeVersions" is not null
         And the session payload field "device.totalMemory" equals 999
+
+     Scenario: On Send Native Callback
+        When I run the "Java Background Crash" mobile scenario
+        And I wait for 4 seconds
+        And I relaunch the Unity mobile app
+        When I run the "On Send Native Callback" mobile scenario
+        Then I wait to receive an error
+
+        And the error payload field "apiKey" equals "Custom ApiKey"
+
+        # misc data
+        And the event "context" equals "Custom Context"
+        And the event "severity" equals "info"
+        And the event "unhandled" is false
+
+        # metadata
+        And the event "metaData.test.scoop" equals "dewoop"
+        And the event "metaData.test2" is null
+
+        # user
+        And the event "user.id" equals "1"
+        And the event "user.email" equals "2"
+        And the event "user.name" equals "3"
+
+        # App Data
+        And the event "app.binaryArch" equals "Custom BinaryArch"
+        And the event "app.buildUUID" equals "Custom BuildUuid"
+        And the event "app.codeBundleId" equals "Custom CodeBundleId" 
+        And the event "app.id" equals "Custom Id"
+        And the event "app.releaseStage" equals "Custom ReleaseStage"
+        And the event "app.type" equals "Custom Type"
+        And the event "app.version" equals "Custom Version"
+        And the event "app.versionCode" equals 999
+        And the event "app.duration" equals 1000
+        And the event "app.durationInForeground" equals 1000
+        And the event "app.inForeground" is false
+        And the event "app.isLaunching" is false
+
+        # Device data
+        And the error payload field "events.0.device.cpuAbi" is an array with 2 elements
```

Comment:
Minor nit - but this might fail if we change the device on which the scenario is run. I think it's fine for current purposes though.

Reply:
I set the array manually to 2 elements